ansible/files/runtimes.json

```diff
@@ -71,7 +71,7 @@
                 "image": {
                     "prefix": "openwhisk",
                     "name": "python2action",
-                    "tag": "latest"
+                    "tag": "1.13.0-incubating"
```

Comment:
Note:  build of python2 image was disabled back in January by https://github.com/apache/incubator-openwhisk-runtime-python/pull/48.  Could create a nightly tag, but probably clearer to use the release tag to indicate we stopped building this image.